Dockerfile

```
@@ -10,7 +10,7 @@ COPY prisma.config.ts .
 
 # Install dependencies based on the preferred package manager
 
-COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml\* .
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml\* .npmrc .
```

Hadolint (2.15.1)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)